src/platforms/alexa/AlexaPlatform.ts

```diff
@@ -137,7 +137,7 @@ export class AlexaPlatform extends VoxaPlatform {
   }
 
   protected getReply(event: IVoxaEvent): IVoxaReply {
-    return new AlexaReply();
+    return new AlexaReply(event);
```

Comment:
i was just thinking, what about instead of having a lot of checks for the type inside the AlexaReply, what about actually having a separate AlexaCanfulfillReply? something that implements the IVoxaReply interface but it's an actually different class

Reply:
I agree, I'll work on implement that

Reply:
@armonge Please check out the latest changes